Add files via upload

StephenPeng111 opened the pull request:
Add sanitized txt files in 4 different languages without sensitive words

StephenPeng111 committed:
Add files via upload

Add sanitized txt file without sensitive words

mj-mueller commented:
Please put the filtered books into `src/main/resources/contentfiles`. Additionally I would need you to filter the nouns, and sentences properties (comdagen/src/main/resources/contentfiles/sentences_de.properties etc.). Also it seems like you need to sign the salesforce cla via https://cla.salesforce.com/sign-cla?prUrl=https%3A%2F%2Fgithub.com%2Fforcedotcom%2Fcomdagen%2Fpull%2F43

mj-mueller committed:
chore: relocate sanitized book files to resource directory

The previous commit added books_{de,generic,ru,zh}.txt at the repo
root; move them under src/main/resources/contentfiles/ so they are
picked up as classpath resources, replacing the prior unsanitized
versions.
